app/controllers/api_controller/collection_config.rb

```diff
@@ -0,0 +1,110 @@
+class ApiController
+  class CollectionConfig < Config::Options
+    def collection?(collection_name)
+      c(collection_name)[:options].include?(:collection)
+    end
+
+    def custom_actions?(collection_name)
+      cspec = c(collection_name)
+      cspec && cspec[:options].include?(:custom_actions)
```

Comment:
Is there a reason we're being defensive about `cspec` here but not elsewhere, such as in the next two methods? Perhaps you could DRY up checking for options?